airflow/models/baseoperator.py

```diff
@@ -1345,6 +1347,10 @@ def get_direct_relatives(self, upstream: bool = False) -> Iterable["DAGNode"]:
     def __repr__(self):
         return "<Task({self.task_type}): {self.task_id}>".format(self=self)
 
+    @property
+    def get_operator_name(self):
+        return self.operator_name
+
```

Comment:
... and we could implement this as


```python
def get_operator_name(self) -> str:
    return self.operator_name or type(self).__name__
```

p.s. when I use `property` I’d drop `get_`, and vice versa.